src/templates/annotation_properties.tsv

```diff
@@ -13,3 +13,4 @@ OMO:0003009	owl:AnnotationProperty	legal name	A synonym type for the legal entit
 OMO:0003010	owl:AnnotationProperty	International Nonproprietary Name	The International Nonproprietary Name (INN) is a standardize name for a pharmaceutical drug or active ingredient issued by the World Health Organization (WHO) meant to address the issues with country- or language-specific brand names. These are issued in several languages, including English, Latin, French, Russian, Spanish, Arabic, and Chinese.	oboInOwl:SynonymTypeProperty	https://orcid.org/0000-0003-4423-4370	2023-09-30	"CHEBI:46195 has been assigned the english International Nonproproprietary Name (INN) ""paracetamol"". In some cases such as this one, the INN might be the same as the ontology's primary label"	https://github.com/information-artifact-ontology/ontology-metadata/issues/149	INN
 OMO:0003011	owl:AnnotationProperty	latin term	A synonym type for describing Latin term synonyms.	oboInOwl:SynonymTypeProperty	https://orcid.org/0000-0002-1773-2692	2023-10-12	nasopharynx (UBERON:0001728) has the latin name "pars nasalis pharyngis"	https://github.com/information-artifact-ontology/ontology-metadata/issues/146
 OMO:0003012	owl:AnnotationProperty	acronym	A synonym type for describing abbreviations that are a part of the full name's words, such as initialisms or alphabetisms.	oboInOwl:SynonymTypeProperty	https://orcid.org/0000-0003-4423-4370	2023-11-01	NASA is an word acronym for the US National Aeronautics and Space Administration because the acronym is pronounced. FBI is an initialism (also known as alphabetism) for the US Federal Bureau of Investigation since the letters are pronounced one at a time. JPEG is an acronym for Joint Photographic Experts Group but does not count as a word acronym nor an initialism since it is mixed how it is pronounced.	https://github.com/information-artifact-ontology/ontology-metadata/issues/135
+oboInOwl:hasDbXref	owl:AnnotationProperty	has database cross reference	An annotation property that links an ontology entity to a database identifier.		https://orcid.org/0000-0002-7356-1779	2024-03-18	disease characteristic (MONDO:0021125) has database cross reference (http://www.geneontology.org/formats/oboInOwl#hasDbXref) "NCIT:C41009"^^xsd:string	https://github.com/information-artifact-ontology/ontology-metadata/issues/123
```

Comment:
a few things I would like clarified in the description:


1. Can only ontology terms be used as the subject?
2. Can ontology terms be used as the object (we see this pretty consistently in OBO foundry ontologies, so I think the answer is yes and we should update the phrase "database cross reference"
3. Can URIs be used as the object or should only `xsd:string`? What about `xsd:QName`, or some other datatype that describes CURIEs?

Reply:
```suggestion
oboInOwl:hasDbXref	owl:AnnotationProperty	has cross-reference	An annotation property that links an ontology entity or a statement to a prefixed identifier.		https://orcid.org/0000-0002-7356-1779	2024-03-18	disease characteristic (MONDO:0021125) has cross-reference (http://www.geneontology.org/formats/oboInOwl#hasDbXref) "NCIT:C41009"^^xsd:string	https://github.com/information-artifact-ontology/ontology-metadata/issues/123
```